### ISSUE TITLE
text on the F1 help screen is difficult to read--glyphs are stretched without anti-aliasing

### ISSUE THREAD

**Issue by xandark** [linappleii/linapple#178]
Can anti-aliasing be used to draw this screen? It's hard to read and doesn't put a good face on this nice project:
![Screenshot 2021-10-05 (19∶03∶50)](https://user-images.githubusercontent.com/5048913/136129193-2e914a19-0b9c-405c-8f8d-ee37cdc7f82c.png)
?

**Comment by ghedger** [comment]
I actually had it working with the Apple II character font at one point.  It looked pretty good, nice and sharp, not ugly, with a size ratio of 1:1, 2:1, 4:1 -- no fractions and no anti-aliasing really needed.  Somehow that work never made it to the main repository and even mine is jacked up again.  

As to anti-aliasing that possibly wades into the territory of the display engine.  SDL 1.2 is limited -- there might be a way to do it, dunno, might not be.  All the Apple II screen rendering is done in software, no GPU.  That's how it was originally done and how it currently stands.

In the interest of efficiency it's been my practice to keep a screen resolution that matches a multiple of 280 in the x and 192 in the y -- 1120 x 768 looks very natural and is a nice native PC resolution.  Maybe not the "modern" way to do it with GPUs nowadayas, but easy and fits the Apple II graphics nicely.

I don't have time to look at this for the foreseeable future (real job, real bills to pay, etc.) but I agree with your suggestion.  If you want to have a look it's in Frame.cpp.  Not my code, desperately needs a re-factor if you've got the spare cycles.

Anyhow, those are some random thoughts.

**Comment by maxolasersquad** [comment]
My personal preference is to use the Apple 2 font here. I have not dug into this at all so I don't know what it is currently using. Maybe it could default to the Apple 2 font, but the config would support specifying any locally installed font.

**Comment by xandark** [comment]
Yeah, that would look nice. Anything that's better than it is now.

**Comment by maxolasersquad** [comment]
There's a few other improvement here too. This is far less grating on þe ole eyeballs.

<img width="902" height="617" alt="Image" src="https://github.com/user-attachments/assets/fab0d5e2-3c9c-4e9d-af4f-bce30419e516" />

**Comment by xandark** [comment]
Great!
